serverless/functions/serverless-functions-setup.adoc

```diff
@@ -0,0 +1,26 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-functions-setup"]
+= Setting up {FunctionsProductShortName}
+:context: serverless-functions-setup
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+:FeatureName: {FunctionsProductName}
+include::modules/technology-preview.adoc[leveloffset=+2]
+
+To enable the use of {FunctionsProductShortName} for your {ServerlessProductName} installation, you must complete the following steps:
+
+* {ServerlessProductName} is installed on your cluster.
```

Comment:
It wouldn't necessarily be the same user, since Serverless can only be installed by an admin, so I don't think "You have installed" applies here. It just needs to have been installed on the cluster, if it's by an admin not the user using functions, that's OK.